### PR TITLE
Scripts and readme

### PR DESCRIPTION
**Pull request by pgier**
Minor improvements to readme.  Try to fix the race conditions in the quickstart deployment example.

### CODE REVIEW

**Comment by pgier** [comment]
Ah, apparently these are all generated, so I need to update the jsonnet source

File: README.md [hunk 1]
```diff
@@ -71,16 +71,16 @@ This adapter is an Extension API Server and Kubernetes needs to be have this fea
...
-$ minikube delete && minikube start --kubernetes-version=v1.14.4 --memory=4096 --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.address=0.0.0.0 --extra-config=controller-manager.address=0.0.0.0
+$ minikube addons disable metrics-server
 ```
 
-The kube-prometheus stack includes a resource metrics API server, like the metrics-server does. So ensure the metrics-server plugin is disabled on minikube:
+In order to just try out this stack, start [minikube](https://github.com/kubernetes/minikube) with the following command:
 
 ```shell
-$ minikube addons disable metrics-server
+$ minikube delete && minikube start --kubernetes-version=v1.14.4 --memory=4g --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.address=0.0.0.0 --extra-config=controller-manager.address=0.0.0.0
```
Comment (metalmatze) [inline README.md]: When touching this, can we already run Kubernetes 1.16?

File: examples/kustomize.jsonnet [hunk 2]
```diff
@@ -6,8 +6,12 @@ local kp =
...
 
 local manifests =
-  { ['00namespace-' + name]: kp.kubePrometheus[name] for name in std.objectFields(kp.kubePrometheus) } +
-  { ['0prometheus-operator-' + name]: kp.prometheusOperator[name] for name in std.objectFields(kp.prometheusOperator) } +
+  { ['setup/0namespace-' + name]: kp.kubePrometheus[name] for name in std.objectFields(kp.kubePrometheus) } +
+  {
+    ['setup/prometheus-operator-' + name]: if name != 'serviceMonitor' then kp.prometheusOperator[name]
+    for name in std.objectFields(kp.prometheusOperator)
+  } +
+  { 'prometheus-operator-serviceMonitor': kp.prometheusOperator.serviceMonitor } +
```
Comment (metalmatze) [inline examples/kustomize.jsonnet]: Can we make sure that the formatting here is pretty much the same compared to `example.jsonnet`?

File: README.md (hunk 1 above)
Comment (pgier) [reply README.md]: yes, I think 1.16 should be fine, I'll update the readme and scripts

File: examples/kustomize.jsonnet (hunk 2 above)
Comment (pgier) [reply examples/kustomize.jsonnet]: jsonnet fmt seemed to be doing different things in one file vs. the other, but I'll try to make the two match

File: README.md [hunk 3]
```diff
@@ -90,20 +90,24 @@ $ minikube addons disable metrics-server
...
 ```shell
-$ kubectl create -f manifests/
+# create namespace, permissions, and CRDs
+kubectl create -f manifests/setup
+
+# wait for CRD creation to complete
+until kubectl get servicemonitors --all-namespaces ; do date; sleep 1; echo ""; done
+
+# create monitoring components
+kubectl create -f manifests/
```
[the remark this answers is not in the document]
Comment (pgier) [reply README.md]: When I apply all the manifests in a single command like `kubectl create -f manifests/setup -f manifests` it seems to consistently fail to create the service monitors.  I guess I can note that due to the race conditions you'll most likely need to run the command more than once.

**Comment by pgier** [comment]
Rebased and squashed commits, and improved commit log to make this a bit more readable.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (pgier) [reply README.md]: I made a couple minor updates to the README to hopefully clarify this.

**Comment by pgier** [comment]
@metalmatze I think this is ready to merge

File: README.md [hunk 4]
```diff
@@ -71,13 +71,13 @@ This adapter is an Extension API Server and Kubernetes needs to be have this fea
 
 ### minikube
 
-In order to just try out this stack, start [minikube](https://github.com/kubernetes/minikube) with the following command:
+To try out this stack, start [minikube](https://github.com/kubernetes/minikube) with the following command:
 
 ```shell
-$ minikube delete && minikube start --kubernetes-version=v1.14.4 --memory=4096 --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.address=0.0.0.0 --extra-config=controller-manager.address=0.0.0.0
+$ minikube delete && minikube start --kubernetes-version=v1.16.0 --memory=4g --bootstrapper=kubeadm --extra-config=kubelet.authentication-token-webhook=true --extra-config=kubelet.authorization-mode=Webhook --extra-config=scheduler.address=0.0.0.0 --extra-config=controller-manager.address=0.0.0.0
```
Comment (LiliC) [inline README.md]: Personally, this was not enough resources for me, its enough if I run it, but it cannot reschedule another pod :D 

But I guess its good enough to test it?

Comment (pgier) [reply README.md]: I can bump it to 6g if you think that's better?

**Pull request merged**